pkg/jwt/parser.go

```diff
@@ -0,0 +1,43 @@
+package jwt
+
+import (
+	"errors"
+
+	"github.com/dgrijalva/jwt-go"
+)
+
+const APPLICATION_TYPE = 2
+
+// TokenClaims represents the role claims presents in the JWT token. This structure extends
```

Comment:
Typo :smile: 

```suggestion
// TokenClaims represents the role claims present in the JWT token. This structure extends
```

There's also a typo in the commit message:

```text
....
type of token has a different way to store the information of whom has
**create** the token:
```
>**create** -> **created**

Reply:
Done.